README.md

```diff
@@ -11,54 +11,437 @@
+- /proxy/transactions
+
+### CLI Summary
+
+```console
+Usage: cardano-wallet COMMAND
+  Cardano Wallet Command-Line Interface (CLI)
+
+Available OPTIONS:
+  -h | --help
+
+Available COMMANDS:
+  launch                Launch and monitor a wallet server and its chain producers.
+  serve                 Serve an HTTP API that listens for commands/actions.
+  mnemonic
+    generate            Generate BIP-39 mnemonic words
+    reward-credentials  Derive reward account private key from mnemonic.
+  wallet
+    list                List all known wallets
+    create
+      from-mnemonic     Create a new wallet using a mnemonic
+      from-public-key   Create a wallet using a public account key
+    get                 Fetch a particular wallet
+    utxo                Get a wallet's UTxO distribution
+    update
+      passphrase        Update a wallet's master passphrase
+      name              Update a wallet's name
+    delete              Forget a wallet and its metadata
+  transaction
+    create              Create a transaction from a known wallet
+    fees                Estimate fees for a transaction
+    list                List the transactions associated with a wallet
+    submit              Submit an externally-signed transaction
+    forget              Forget a pending transaction with specified id
+  address
+    list                List all known addresses of a wallet
+  stake-pool
+      list              List all known stake pools
+  network
+    information         View network information
+    parameters          View network parameters
+    clock               View NTP offset
+  key
+    root                Extract root extended private key from a mnemonic sentence.
+    child               Derive child keys.
+    public              Extract public key from a private key.
+    inspect             Show information about a key.
+  version               Show the program's current version
+```
+
+## Testing
+
+See [Wiki - Testing](https://github.com/input-output-hk/cardano-wallet/wiki/Testing)
+
+
+## Examples
+
+### Create a new Address from mnemonic
+
+Generate BIP-39 mnemonic words using the CLI via docker exec.
+
+```console
+$ docker exec cardano-wallet_cardano-wallet_1 cardano-wallet-byron mnemonic generate
+effort wink follow warm utility brush series rubber hawk antenna squirrel mystery wise village palace
+```
+
+Note that in the above example "cardano-wallet_cardano-wallet_1" is the name of the running container. If this is not the container name for you, use `docker ps` to find the container name for cardano-wallet and replace in the example.
+
+### Restore mnemonic to wallet
+
+```console
+$ curl -H "Content-Type: application/json" -d '{"name":"BobsTestWallet","mnemonic_sentence":["effort","wink","follow","warm","utility","brush","series","rubber","hawk","antenna","squirrel","mystery","wise","village","palace"],"passphrase":"MyVerySecurePassphrase","style":"ledger"}' -X POST http://localhost:8090/v2/byron-wallets
```

Comment:
Examples should be consistent, if the previous one was using the command-line, this one should too.